src/main/java/org/dependencytrack/tasks/metrics/ComponentMetricsUpdateTask.java

```diff
@@ -122,29 +121,36 @@ static Counters updateMetrics(final UUID uuid) throws Exception {
                 }
 
                 switch (Policy.ViolationState.valueOf(violation.violationState().name())) {
-                    case FAIL -> counters.policyViolationsFail++;
-                    case WARN -> counters.policyViolationsWarn++;
-                    case INFO -> counters.policyViolationsInfo++;
+                    case FAIL -> counters.policyViolationsFailTotal++;
+                    case WARN -> counters.policyViolationsWarnTotal++;
+                    case INFO -> counters.policyViolationsInfoTotal++;
                 }
-            }
 
-            if (counters.policyViolationsLicenseTotal > 0) {
-                counters.policyViolationsLicenseAudited = toIntExact(getTotalAuditedPolicyViolations(pm, component, PolicyViolation.Type.LICENSE));
-                counters.policyViolationsLicenseUnaudited = counters.policyViolationsLicenseTotal - counters.policyViolationsLicenseAudited;
-            }
-            if (counters.policyViolationsOperationalTotal > 0) {
-                counters.policyViolationsOperationalAudited = toIntExact(getTotalAuditedPolicyViolations(pm, component, PolicyViolation.Type.OPERATIONAL));
-                counters.policyViolationsOperationalUnaudited = counters.policyViolationsOperationalTotal - counters.policyViolationsOperationalAudited;
-            }
-            if (counters.policyViolationsSecurityTotal > 0) {
-                counters.policyViolationsSecurityAudited = toIntExact(getTotalAuditedPolicyViolations(pm, component, PolicyViolation.Type.SECURITY));
-                counters.policyViolationsSecurityUnaudited = counters.policyViolationsSecurityTotal - counters.policyViolationsSecurityAudited;
+                if (BooleanUtils.isTrue(violation.suppressed)) {
```

Comment:
I think this should check for `BooleanUtils.isFalse(violation.suppressed)` instead. Suppressed findings and violations are counted separately and are not included in `*Audited` metrics.